src/test/java/org/mule/extension/http/api/request/validator/ExpressionResponseValidatorTestCase.java

```diff
@@ -75,14 +76,21 @@ public void expressionEvaluatingToFalse() throws Exception {
   }
 
   @Test(expected = IllegalStateException.class)
-  public void whenLiteralExpressionIsNotPresentTheValidatorThrowsException() {
+  public void whenLiteralExpressionIsNotPresentTheValidatorThrowsException() throws NoSuchFieldException, IllegalAccessException {
     ExpressionResponseValidator validator = new ExpressionResponseValidator();
     Literal<String> mockLiteral = mock(Literal.class);
     when(mockLiteral.getLiteralValue()).thenReturn(empty());
-    validator.setExpression(mockLiteral);
+    setExpression(validator, mockLiteral);
```

Comment:
ideally you should do this with MTF and not require setter at all. But it you have no choice than to do this, then leave the setter but with package scope